lxd/device/config/device_proxyaddress.go

```diff
@@ -2,8 +2,9 @@ package config
 
 // ProxyAddress represents a proxy address configuration.
 type ProxyAddress struct {
-	ConnType string
-	Abstract bool
-	Address  string
-	Ports    []uint64
+	ConnType   string
+	Abstract   bool
+	Address    string
+	Ports      []uint64
```

Comment:
@markylaing can we remove the `Ports` field and just have `PortRanges`, and update where uses `Ports` to expand if needed.

Reply:
I have removed `PortRanges` in favour of using the original `Ports` field. See https://github.com/lxc/lxd/pull/9614#discussion_r762862079.